src/request/component-models/LoggingComponentModel.ts

```diff
@@ -20,6 +22,31 @@ export class LogMessageModel implements ILogMessageModel {
         return this._message.id;
     }
 
+    public get isObject(): boolean {
+        
+        //
+        // NOTE: Our heuristic for determining whether text represents an object rather simplistic 
+        //       (to minimize impact on performance). We simply look for starting and ending braces
+        //       (i.e. '{' and '}') near the beginning and ending of the text, respectively, allowing 
+        //       for whitespace and other text that might pre-fix/post-fix the actual object.
+        //
```

Comment:
I've also updated the property so that the heuristic is only ever executed once per model.